server/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -321,3 +321,7 @@ tracing.datadog.timeout=10s
 tracing.stackdriver.enabled=false
 tracing.stackdriver.timeout=30s
 tracing.stackdriver.gcpProjectId=
+
+# Cors config
+nuxeo.cors.allowSubdomains=true
+nuxeo.cors.supportedMethods=GET,POST,HEAD,OPTIONS,DELETE,PUT
```

Comment:
I think it's worth also adding an empty

    nuxeo.cors.urls=

to make explicit what can be configured. You may want to change the comment to `# CORS config (active if nuxeo.cors.urls has a value)`

Then in the nxftl file you can test presence with `<#if (nuxeo.cors.urls)?has_content>`